doc/contribute/external.rst

```diff
@@ -164,3 +170,33 @@ The flowchart below shows an overview of the process:
    :align: center
 
    Submission process
+
+.. _external-tooling:
+
+Contributing External Tooling
+*****************************
+
+This section deals exclusively with the inclusion of external tooling in the
+Zephyr project, where tooling is defined as software that assists the
+compilation, testing or simulation processes but in no case ends up being part
+of the code compiled and linked into the final image.
+
+There are no license requirements for tooling components, but licenses that
+have not been approved by the `Open Source Initiative (OSI)`_ might be rejected.
```

Comment:
This feels a little strange. If there are no license requirements, why would a tool component be rejected due to its license?